frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -24,13 +24,13 @@ page-section.c-section(:title='L("Invite links")')
         )
     tbody
       tr(
-        v-for='(item, index) in activeList'
-        :key='index'
+        v-for='(item, index) in invitesToShow'
+        :key='item.inviteSecret'
       )
         td.c-name
-          | {{ item.name }}
+          | {{ item.invitee }}
           tooltip.c-name-tooltip(
-            v-if='item.isAnyoneLink'
+            v-if='false'
```

Comment:
Why is there a `v-if` if it's always false?

Reply:
For now it's false because it's missing the implementation of "Anyone link", that will be done in a next PR after #747 is merged

Reply:
Gotcha! OK, let's wait for #747 to be merged then, especially because of the other changes to group.js